src/amber/cli/templates/app/src/views/home/index.slang.ecr

```diff
@@ -1,8 +1,9 @@
-div#logo.col-sm-6
-div.col-sm-6
-  h2 = t "welcome_to_amber"
-  p Thank you for trying out the Amber Framework.  We are working hard to provide a super fast and reliable framework that provides all the productivity tools you are used to without sacrificing the speed.
-  div.list-group
-    a.list-group-item.list-group-item-action target="_blank" href="https://docs.amberframework.org" Getting Started with Amber Framework
-    a.list-group-item.list-group-item-action target="_blank" href="https://github.com/veelenga/awesome-crystal" List of Awesome Crystal projects and shards
-    a.list-group-item.list-group-item-action target="_blank" href="https://crystalshards.xyz" What's hot in Crystal right now
+div.row
```

Comment:
You do not have to define the `div` here `.row` will render the div by default. this will clean up the template

Reply:
good point!